dokka-runners/dokka-gradle-plugin/src/test/kotlin/DokkaPluginTest.kt

```diff
@@ -115,5 +120,74 @@ class DokkaPluginTest : FunSpec({
                 }
             }
         }
+
+        context("DokkaPublications") {
```

Comment:
nit: maybe it makes to test that publications have a default without extension level suppressions since this is a [part of the behavior](https://github.com/Kotlin/dokka/pull/3830/files#diff-55ceb7738ade299cded228f9d44aa544238c6afaf5ccb2c5e59b2b41717325abR164)

Reply:
These properties are required, and all tests will fail if they are unset.

https://github.com/Kotlin/dokka/blob/e3dc412e9185564a49ecc639a7291b0596abc0a0/dokka-runners/dokka-gradle-plugin/src/main/kotlin/engine/parameters/builders/DokkaParametersBuilder.kt#L46-L47